frontend/Common/ModalWindow.svelte

```diff
@@ -19,10 +20,10 @@
     }
 </script>
 
-<svelte:window onkeydown={handleKeyDown} />
+<svelte:window onkeydown={handleKeyDown} bind:innerWidth={innerWidth} />
 <div class="modal-window">
     <div class="d-flex align-items-center justify-content-center p-4">
-        <div class="rounded bg-white p-4 {widthClass}">
+        <div class="rounded bg-white p-4 {innerWidth >= 768 ? widthClass : "w-100"}">
```

Comment:
why not use [bootstrap](https://getbootstrap.com/docs/5.3/layout/containers/) containers?

Reply:
ok, lets keep it as is